### PR TITLE
Regression test for dyn discadj fsi

### PR DESCRIPTION
**Pull request by cvencro**
Files for new regression test added to support PR [#1260](https://github.com/su2code/SU2/pull/1260).

### CODE REVIEW

**Review by pcarruscag — APPROVED** [review]

**Comment by pcarruscag** [comment]
The restart files would take a little less space if they were binary.

File: disc_adj_fsi/dyn_fsi/grad_dv.opt.ref [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INDEX	GRAD
+0	-3.465631474226565e-03
```
Comment (TobiKattmann) [inline disc_adj_fsi/dyn_fsi/grad_dv.opt.ref:2]: This is based on the temporary output, so it might change still right?

Comment (pcarruscag) [reply disc_adj_fsi/dyn_fsi/grad_dv.opt.ref:2]: Good point. It is more convenient to add the reference results to the code repo instead.

Comment (cvencro) [reply disc_adj_fsi/dyn_fsi/grad_dv.opt.ref]: Reference switched to the code repo.

**Review by TobiKattmann — APPROVED** [review]

**Pull request merged**